README.md

```diff
@@ -1,44 +1,154 @@
+
+`config.json`:
 ```
-$ npm install
+{
+  "discoveryUsername" : "YOUR_WATSON_DISCOVERY_USERNAME",
+  "discoveryPassword" : "YOUR_WATSON_DISCOVERY_PASSWORD"
+}
 ```
 
-Next, provide your Cloudant URL from your Cloudant Service Credentials as an environment variable.
+Create configmap in Kubernetes for `config.json`
 
 ```
-export CLOUDANT_URL=https://1111405-438f-4374-9b43-975e0220f3cd-bluemix:935684a91478c5f8918d3d0b2e004ef6fe825939ed2304f28ec2225fa9a952df@8bb72405-438f-4374-9b43-975e0220f3cd-bluemix.cloudant.com
+$ kubectl create configmap watson-discovery-config --from-file=config.json
 ```
-Now, you are ready to run the app!
+
+### 5. Deploy the application
+<!--Deploy in kubernetes (1) nodejs app and (2) couchdb instance. Access via external ip-->
+
+Deploy the CouchDB database. This is where the reviews and Watson Discovery results will be stored.
+
+```
+$ kubectl apply -f couchdb.yaml
+```
+
+Deploy the Watson Review Analyzer app.
+
 ```
-$ node app.js
+$ kubectl apply -f watson-review-analyzer.yaml
 ```
 
-## 3. Usage
-`curl http://localhost:8080/reviews/[product-id]`
+You can access the application in the browser via the Load Balancer's IP (External IP).
+If you don't have the Load Balancer enabled, you can access it through one of your Kubernetes' worker IP and service Node port of watson-review-analyzer.
+
+<pre>
+$ kubectl get service
+
+NAME             CLUSTER-IP       EXTERNAL-IP      PORT(S)          AGE
+couchdb          172.21.174.52    169.xy.xyz.220   5984:32402/TCP   13d
+kubernetes       172.21.0.1       &ltnone&gt           443/TCP          27d
+watson-reviews   172.21.254.246   <b>169.48.xyz.221</b>   80:<b>31385</b>/TCP     15m
+</pre>
+
+If you don't have the Load Balancer and need the worker IP:
+<pre>
+$ bx cs workers YOUR_CLUSTER_NAME
+
+OK
+ID                                                 Public IP       Private IP       Machine Type   State    Status   Zone    Version
+kube-dal12-cr5c30966926aa444f9c02c72d5c3e1ca5-w1   <b>169.xy.xyz.35</b>   10.184.120.196   b2c.16x64      normal   Ready    dal12   1.8.6_1506*
+</pre>
+
+Go to `169.xy.xyz.221` or `169.xy.xyz.35:31385` (for clusters without a Load Balancer)
+
+![Landing Page](docs/landing-page.png)
+
+### 6. Search for a product in Amazon
+
+To use the app, search for the product you want the reviews to be uploaded to Watson Discovery and get its product ID (ASIN).
+
+`https://www.amazon.com/Samsung-Thinnest-Premium-Anti-Scratch-Protective/dp/`**`B06XZ2CM2H`**`/ref=cm_cr_arp_d_product_top?ie=UTF8`
+
+In this case, the product ID is **B06XZ2CM2H**
+
+![Landing Page](docs/input.png)
+
+Go ahead and press `Go`.
+
+After Watson Discovery finishes processing all the reviews, the app should show you its General Sentiment and Top entities found.
 
-example:
+![Landing Page](docs/discovery-result.png)
 
-`https://www.amazon.com/Colorful-Ultra-Thin-Anti-Drop-Premium-Material/product-reviews/B06XZ2CM2H/ref=cm_cr_getr_d_paging_btm_1?ie=UTF8&pageNumber=1`
+# Learn more
```

Comment:
please add these too:

<pre>
# Links
* [Demo on Youtube](https://youtu.be/rVL1HsbsdBI)
* [Watson Node.js SDK](https://github.com/watson-developer-cloud/node-sdk)

# Learn more

* **Artificial Intelligence Code Patterns**: Enjoyed this Code Pattern? Check out our other [AI Code Patterns](https://developer.ibm.com/code/technologies/artificial-intelligence/).
* **AI and Data Code Pattern Playlist**: Bookmark our [playlist](https://www.youtube.com/playlist?list=PLzUbsvIyrNfknNewObx5N7uGZ5FKH0Fde) with all of our Code Pattern videos
* **With Watson**: Want to take your Watson app to the next level? Looking to utilize Watson Brand assets? [Join the With Watson program](https://www.ibm.com/watson/with-watson/) to leverage exclusive brand, marketing, and tech resources to amplify and accelerate your Watson embedded commercial solution.
</pre>